guides/doc-Configuring_Ansible/master.adoc

```diff
@@ -7,9 +7,10 @@ endif::[]
 
 include::common/attributes.adoc[]
 
-:configuring-ansible:
 :context: ansible
 
+:configuring-ansible:
```

Comment:
All looks good, In only have a question to this change. Why you do this? Won't it break something?

Reply:
I had a problem, and was troubleshooting, comparing the layout of the Installation docs master pages to what i had done with this new guide. I moved things around trying to figure out what was wrong. This was completely unrelated in the end. I will move this back. Thanks for catching.